floppyforms/widgets.py

```diff
@@ -12,7 +12,13 @@
 from django.forms.widgets import FILE_INPUT_CONTRADICTION
 from django.conf import settings
 from django.template import loader
-from django.utils.datastructures import MultiValueDict, MergeDict
+try:
+    from django.utils.datastructures import MultiValueDict, MergeDict
+    MULTIVALUE_DICT_TYPES = (MultiValueDict, MergeDict)
+except ImportError:
+    # Django >= 1.9
+    from django.utils.datastructures import MultiValueDict
+    MULTIVALUE_DICT_TYPES = (MultiValueDict,)
```

Comment:
I like to keep logic that forks on different Django versions in a `compat.py` file. We should stick this try/except in this file: https://github.com/gregmuellegger/django-floppyforms/blob/master/floppyforms/compat.py